tests/test_wake.py

```diff
@@ -1,47 +1,167 @@
 import torch
+from torch import nn
+from torch.distributions.normal import Normal
+
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
I think the test function is supposed to take `devices` as an argument so that the tests can run on any gpu (not just `cuda:0`). That seems to be how it's done in some of the other tests.

Reply:
Oh, you did that. Do we need the line above then?